components/StoryCard.tsx

```diff
@@ -17,62 +19,87 @@ const borderColors = {
   release: 'blue',
 };
 
+const StyledSpan = styled.span`
+  margin-left: 4px;
+  cursor: pointer;
+  color: #3291ff;
+`;
+
 interface StoryCardParams {
   story: Story;
   index: number;
+  state: string;
   addFilter: (name: string, filter: Owner | Label | Iteration) => void;
   openStory: (story: Story) => void;
 }
 
-const StoryCard = ({ story, index, addFilter, openStory }: StoryCardParams): JSX.Element => {
+const StoryCard = ({ story, state, index, addFilter, openStory }: StoryCardParams): JSX.Element => {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+
+  const openEstimationModal = (e): void => {
+    e.stopPropagation();
+    setIsModalVisible(true);
+  };
   return (
-    <Draggable key={story.id} draggableId={story.id.toString()} index={index}>
-      {(provided: Draggable.provided) => (
-        <div
-          style={{ background: 'green' }}
-          {...provided.draggableProps}
-          {...provided.dragHandleProps}
-          ref={provided.innerRef}
-        >
-          <CardContainer
-            width="250px"
-            hoverable
-            color={borderColors[story.story_type] || 'gray'}
-            onClick={() => openStory(story)}
+    <>
```

Comment:
Will we get a missing key error here on iterarting?

Reply:
The console is clean of missing keys errors. I also removed it from the `Draggable` since it is explicitly set inside the `Column` component `.map` on every`StoryCard` component instance.